src/user/userRoutes.js

```diff
@@ -1,8 +1,12 @@
 const { Router } = require("express");
-const { addUser } = require("./userController");
+const { addUser, deleteUser } = require("./userController");
 const { hashPassword, checkPassword, checkEmail } = require("../middleware");
 const userRouter = Router();
 
-userRouter.post("/user", addUser);
+userRouter.post("/user", checkEmail, hashPassword, addUser);
+userRouter.delete("/deleteUser", deleteUser);
+userRouter.get("/user", listUsers);
```

Comment:
we're not doing listUsers as discussed earlier, unless this will be used for listing info of the user that is using the app? maybe call it readUser for it to make more sense?